docs/pico_sdk.md

```diff
@@ -139,13 +139,14 @@ then passing the `SPI` object to `RF24::begin(_SPI*)`.
 #include <RF24.h>
 
 RF24 radio(7, 8); // pin numbers connected to the radio's CE and CSN pins (respectively)
+SPI spi_;
 
 int main()
 {
     // again please review the GPIO pins' "Function Select Table" in the Pico SDK docs
-    SPI.begin(spi0, 2, 3, 4); // must use spi0 or spi1 as 1st parameter
+    spi_.begin(spi0, 2, 3, 4); // spi0 or spi1 bus, SCK, TX, RX
```

Comment:
I recently was wondering why I didn't describe the pin parameters better. Thanks for this.